pkg/cmd/openfaas_ingress_app.go

```diff
@@ -0,0 +1,159 @@
+package cmd
```

Comment:
What happened here?

Reply:
I think iv managed to break my fork. Going to close this and re-do